app/queries/check_for_content_item_preventing_draft_from_being_published.rb

```diff
@@ -0,0 +1,53 @@
+module Queries
+  module CheckForContentItemPreventingDraftFromBeingPublished
+    extend ArelHelpers
+
+    # Checks for any content item which would prevent a content item with the
+    # specified content_id and base_path from being published (from the draft
+    # state).
+    def self.call(content_id, base_path, document_type)
+      if SubstitutionHelper::SUBSTITUTABLE_DOCUMENT_TYPES.include?(document_type)
+        nil # The SubstitionHelper will unpublish any item that is in the way
```

Comment:
Talked offline, the "what" is definitely wrong, as this should return nil, and currently it does nothing. I'll fix this, and add a test to cover this case.

Good spot Doug :)